index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WOS Concert</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="head-navbar">
+            <div class="mobile-nav">
+                <nav class="with-hamburger">
+                    <div class="slidediv">
+                        <ul class="olist">
+                            <a class="olist-a" href='about.html'><li>About</li></a>
+                            <a class="olist-a" href="#main-program"><li>Program</li></a>
+                            <a class="olist-a" href="#join-event-button"><li>Join</li></a>
+                            <a class="olist-a" href="#partners"><li>Sponsor</li></a>
+                            <a class="olist-a" href="#"><li>News</li></a>
+                        </ul>
+                    </div>
+                    <button class="menu-button">
+                        <div class="ham">
+                            <div class="bar1"></div>
+                            <div class="bar2"></div>
+                            <div class="bar3"></div>
+                        </div>
+                    </button>
+                </nav>
+            </div>
+            <div class="desktop-nav">
+                <div class="header-bar">
+                    <ul class="header-bar-list">
+                        <li><ion-icon name="logo-facebook"></ion-icon></li>
+                        <li><ion-icon name="logo-twitter"></ion-icon></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+                <nav class="with-logo-list">
+                    <div class="logo">
+                        <a class='logo-img-link' href='index.html'><img class="logo-img" src="./media/logo/WOSresize.png" alt="WOS Logo"></a>
+                    </div>
+                    <div class="nav-list">
+                        <ul class="nav-list-items">
+                            <li><a href='about.html'>About</a></li>
+                            <li><a href='#main-program'>Program</a></li>
+                            <li><a href='#join-event-button'>Join</a></li>
+                            <li><a href='#partners'>Sponser</a></li>
+                            <li><a href=''>News</a></li>
+                        </ul>
+                    </div>
+                    <div class="tour-button">
+                        <button class="wos-tour-button">WOS Tour</button>
+                    </div>
+                </nav>
```

Comment:
**OPTIONAL** - It is not advisable to have two `nav` tags on one HTML page, I suggest you have just one nav tag, then use CSS to customize the nav for either the mobile or desktop version. Please fix this for both pages 🙏